Implemented the `partial_update_class` endpoint

Chris-ade opened the pull request:
I've implemented the `partial_update_class` endpoint using the existing UpdateClassSchema in the schema.py.

Chris-ade committed:
Implemented the `partial_update_class` endpoint

gray-adeyi commented on routers/classes.py:
```diff
@@ -76,8 +77,26 @@ async def get_class_students(
+    for key, value in class_data.dict(exclude_unset=True).items():
+        setattr(class_instance, key, value)
 
+    # Commit the changes to the database
```
You have to add the class to the db before commit. `db.add(class_instance)`

gray-adeyi commented on routers/classes.py:
```diff
@@ -1,6 +1,6 @@
 from uuid import UUID
 
-from fastapi import APIRouter, Depends, status, HTTPException
+from fastapi import APIRouter, Depends, status, HTTPException, Patch
 from sqlalchemy.exc import IntegrityError
```
Was there a reason why Patch was imported?

Chris-ade replied:
It's not necessarily required in the endpoint, sorry for the overlook

gray-adeyi replied:
It's cool. I was only asking. Were you able to set up the project locally. It makes sense to at least try the implementation you're send in since there's no test for it at the moment.

Chris-ade committed:
Updated the `partial_update_class` endpoint

Removed: `Patch` as it's not required in the endpoint.
Added: the code to add the class instance back to the session in the endpoint.
